sdk/servicebus/azure-servicebus/tests/servicebus_preparer.py

```diff
@@ -1,16 +1,22 @@
 import functools
-import hashlib
 import os
 import time
-from collections import namedtuple
+import datetime
+import logging
+
+from azure_devtools.scenario_tests import AzureTestError, ReservedResourceNameError
+try:
+    from azure.mgmt.resource import ResourceManagementClient
+except ImportError:
```

Comment:
without this wont the entire thing just error out as none of the queues etc will be created?

Reply:
yeah we can prob remove it